warehouse/templates/pages/help.html

```diff
@@ -112,16 +112,16 @@ <h2 class="faq-title" id="Basics">Basics</h2>
 
         <h3 id="packages">{{ packages() }}</h3>
         <p>
-          We use a number of terms to describe software available on PyPI, like <em>project</em>, <em>release</em>, <em>file</em>, and <em>package</em>. Sometimes those terms are confusing because they're used to describe different things in other contexts. Here's how we use them on PyPI:
+          We use a number of terms to describe software available on PyPI, like project, release, file, and package. Sometimes those terms are confusing because they're used to describe different things in other contexts. Here's how we use them on PyPI:
```

Comment:
Can we put these in quotes instead? I'd like to maintain some sort of distinction that these are specific terms. (Same for the lines below)